src/routes/pricing/+page.svelte

```diff
@@ -400,6 +416,10 @@
 
     .web-pricing-cards-list {
         gap: 1rem;
+        grid-template-columns: repeat(3, 1fr);
+    }
+
+    :global(.show-scale-plan) .web-pricing-cards-list {
```

Comment:
we can avoid `global` and use a css variable based on the value of the visible plans length like - `--columns-template={...}` and then reference that in css.